pkg/render/logstorage.go

```diff
@@ -584,6 +577,16 @@ func (es elasticsearchComponent) podTemplate() corev1.PodTemplateSpec {
 		})
 	}
 
+	sc := securitycontext.NewRootContext(false)
+	// These capabilities are required for docker-entrypoint.sh.
+	// See: https://github.com/elastic/elasticsearch/blob/7.17/distribution/docker/src/docker/bin/docker-entrypoint.sh.
+	// TODO Consider removing for Elasticsearch v8+.
+	sc.Capabilities.Add = []corev1.Capability{
+		"SETGID",
+		"SETUID",
+		"SYS_CHROOT",
+	}
```

Comment:
Does the ES entrypoint script work without `SETGID` and `SETUID`?

Reply:
Not really. I believe `chroot` needs them.